src/components/list.js

```diff
@@ -0,0 +1,15 @@
+const List = ({ things }) => {
```

Comment:
"thing" is not the best name for an argument. No need to refactor, but keep in mind naming is very important